SimPEG/electromagnetics/static/induced_polarization/run.py

```diff
@@ -47,7 +47,7 @@ def run_inversion(
             mesh,
             active_cells=actind,
             mapping=regmap,
-            cell_weights=mesh.cell_volumes[actind],
```

Comment:
this line should actually just be removed now (don't replace it with anything). As is, it is double counting cell volume.

Reply:
Nice catch!

Reply:
Done